core/services/headreporter/prometheus_reporter.go

```diff
@@ -92,6 +93,9 @@ func (pr *prometheusReporter) reportPendingEthTxes(ctx context.Context, evmChain
 
 	unconfirmed, err := txm.CountTransactionsByState(ctx, txmgrcommon.TxUnconfirmed)
 	if err != nil {
+		if strings.Contains(err.Error(), "disabled") {
```

Comment:
Is it possible to use `errors.Is`?
```suggestion
		if errors.Is(err, disabledErr) {
```

Reply:
Oh it looks like that would requires a framework update

Reply:
- https://github.com/smartcontractkit/chainlink-framework/pull/13